src/sql/connection/connection.py

```diff
@@ -814,24 +823,37 @@ def raw_execute(self, query, parameters=None, with_=None):
                     "Cannot execute query because the following "
                     "variables are undefined: {}".format(", ".join(missing_parameters))
                 )
-
             return self._connection_execute(query, parameters)
         else:
             try:
+                if enable_named_params == "disable":
+                    return self._connection_execute(
+                        query, parameters=enable_named_params
```

Comment:
Here also, it looks like `parameters` is supposed to pass the user namespace . So passing a flag instead looks confusing

Reply:
[Fixed](https://github.com/ploomber/jupysql/pull/980/files#diff-62e3d6289e3242b271e6df3a7f201ab003f8ce73a5daf1a824cc2b4d892513d1R823-R824)